### PR TITLE
Help Suite - Learn More and Api Documentation

### PR DESCRIPTION
**Pull request by fernandoiquiza**

### CODE REVIEW

File: Pivotal-Automation/pages/src/main/java/org/fundacion/pages/help/LearnMorePage.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.awt.*;
+import java.awt.event.KeyEvent;
+import java.util.ArrayList;
+
+/**
```
Comment (DavidVallejos) [inline Pivotal-Automation/pages/src/main/java/org/fundacion/pages/help/LearnMorePage.java:13]: Put your name

File: Pivotal-Automation/automation/pom.xml [hunk 2]
```diff
@@ -58,6 +58,7 @@
                     </properties>
                     <suiteXmlFiles>
                         <suiteXmlFile>src/resource/testsuites/${suite}.xml</suiteXmlFile>
+                        <!--<suiteXmlFile>src/resource/testsuites/helpSuite.xml</suiteXmlFile>-->
```
Comment (DavidVallejos) [inline Pivotal-Automation/automation/pom.xml:61]: Clean your comments

File: Pivotal-Automation/automation/src/resource/testsuites/helpSuite.xml [hunk 3]
```diff
@@ -7,6 +7,8 @@
     <test name="HelpTestClasses">
         <classes>
             <class name="org.fundacion.automation.help.TestVerifyQuickStartPage"/>
+            <class name="org.fundacion.automation.help.TestLearnMore"/>
+            <!--<class name="org.fundacion.automation.help.TestApiDocumentation"/>-->
```
Comment (DavidVallejos) [inline Pivotal-Automation/automation/src/resource/testsuites/helpSuite.xml:11]: Here, clean comment

**Review by DavidVallejos — APPROVED** [review]

File: Pivotal-Automation/pages/src/main/java/org/fundacion/pages/help/LearnMorePage.java (hunk 1 above)
Comment (fernandoiquiza) [reply Pivotal-Automation/pages/src/main/java/org/fundacion/pages/help/LearnMorePage.java:13]: Ok

**Pull request merged**